demo/src/main/scala/io/finch/demo/model.scala

```diff
@@ -35,12 +35,16 @@ object model {
 
   // A ticket object with two fields: `id` and `label`.
   case class Ticket(id: Long, label: String) extends ToJson {
-    override def toJson = Json.obj("id" -> id, "label" -> label)
+    override def toJson = Json.obj("id" -> jNumber(id), "label" -> jString(label))
   }
 
   // A user object with three fields: `id`, `name` and a collection of `tickets`.
   case class User(id: Long, name: String, tickets: Seq[Ticket]) extends ToJson {
-    override def toJson = Json.obj("id" -> id, "name" -> name, "tickets" -> Json.arr(tickets.map(_.toJson) :_*))
+    override def toJson = Json.obj(
```

Comment:
Similarly here:

``` scala
    override def toJson = Json(
      "id" := id,
      "name" := name,
      "tickets" := tickets.map(_.toJson).toList
    )
```

Could also change the `Seq` to `List` and skip the `toList`.